engines/scc_proxy/lib/scc_proxy/engine.rb

```diff
@@ -117,6 +117,17 @@ def prepare_scc_request(uri_path, product, auth, params, mode)
       scc_request
     end
 
+    def prepare_scc_update_request(uri_path, product, auth, mode)
```

Comment:
yes, `prepare_scc_request` and `prepare_scc_update_request` can be refactored into one method

let's get make it work first, then let's make it pretty